pkg/cluster/healthcheck.go

```diff
@@ -20,9 +20,11 @@ import (
 // <random>.<location>.cloudapp.azure.com).  It will then negotiate TLS against
 // the former address (i.e. openshift.<random>.osadev.cloud), verifying that the
 // server certificate presented matches cert.
-func getHealthCheckHTTPClient(dialHost string, cert *x509.Certificate) *http.Client {
+func getHealthCheckHTTPClient(dialHost string, certs []*x509.Certificate) *http.Client {
 	pool := x509.NewCertPool()
-	pool.AddCert(cert)
+	for _, cert := range certs {
```

Comment:
this should just be using the ca certificate - basically the output of the derived function for the input cert in question